src/track/track.h

```diff
@@ -402,6 +406,7 @@ class Track : public QObject {
             mixxx::TrackRecord newRecord,
             mixxx::BeatsPointer pOptionalBeats = nullptr);
 
+    bool isDirty();
```

Comment:
Why did you mark this comment as resolved while keeping the reordering??

Reply:
I had removed the unrelated reorderings, this function is clearly related.
Do you really prefer to keep the files messy rather than grouping functions with some logic, as you don't want separate cleanup PRs either?

Reply:
The reorderings could be done in a separate PR that does not modify the implementations. I already explained why.